src/lib/gridsterDraggable.service.ts

```diff
@@ -243,9 +245,25 @@ export class GridsterDraggable {
       this.mousedown();
       this.touchstart();
     }
+    if (!enableDrag) {
```

Comment:
why this here ?
you are already in dragMove means you are already making a drag. ...